src/txn/txn.c

```diff
@@ -1378,12 +1360,8 @@ __txn_resolve_prepared_op(WT_SESSION_IMPL *session, WT_TXN_OP *op, bool commit,
      * prepared updates are written to the data store. When the page is read back into memory, there
      * will be only one uncommitted prepared update.
      */
-    if (fix_upd != NULL) {
+    if (fix_upd != NULL)
```

Comment:
This function is called only for `commit` now. You can add a check outside and change passing the parameters.

Reply:
Done